src/components/Calculator.jsx

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import '../styles/Calculator.css';
+
+const Calculator = () => (
+  <div className="calculator">
+    <div className="screen">0</div>
+    <div className="buttons">
+      <div className="row">
+        <Button label="AC" color="light-gray" />
+        <Button label="+/-" color="light-gray" />
+        <Button label="%" color="light-gray" />
+        <Button label="÷" color="orange" />
+      </div>
+      <div className="row">
+        <Button label="7" color="gray" />
+        <Button label="8" color="gray" />
+        <Button label="9" color="gray" />
+        <Button label="×" color="orange" />
+      </div>
+      <div className="row">
+        <Button label="4" color="gray" />
+        <Button label="5" color="gray" />
+        <Button label="6" color="gray" />
+        <Button label="-" color="orange" />
+      </div>
+      <div className="row">
+        <Button label="1" color="gray" />
+        <Button label="2" color="gray" />
+        <Button label="3" color="gray" />
+        <Button label="+" color="orange" />
+      </div>
+      <div className="row">
+        <Button label="0" color="gray" doubleSize />
+        <Button label="." color="gray" />
+        <Button label="=" color="orange" />
+      </div>
```

Comment:
- [ ] it would be better if you make the borders between the buttons a bit lighter as in the given design in order to match it 
- yours
<img width="474" alt="Screenshot 2023-05-31 at 22 51 46" src="https://github.com/Jrmcuervo/math-magicians-rc/assets/61790587/14d9a67b-bf0a-4af6-a312-071a13102b3e">

- Design 
<img width="341" alt="Screenshot 2023-05-31 at 22 51 58" src="https://github.com/Jrmcuervo/math-magicians-rc/assets/61790587/e67f0d9a-d78c-4763-ac97-dfc654e7b80b">